tests/expressions/tests.py

```diff
@@ -1146,6 +1146,15 @@ def test_date_subtraction(self):
             expected.add('e1')
         self.assertEqual(less_than_5_days, expected)
 
+    @unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL specific operation")
+    def test_date_nondate_subtraction(self):
+        # The result of (date_field - int_field) varies across backends so test
+        # only on postgres since it returns something sensible - a date.
```

Comment:
FYI - so does Oracle. But I don't think that distinction matters here because we aren't guarding functionality behind the decision, just the test. Generally Django prefers skipping based on features rather than vendors, but this isn't worth a feature flag IMO.